clash-vexriscv-sim/tests/Tests/Jtag.hs

```diff
@@ -38,9 +40,20 @@ getOpenOcdCfgPath = getDataFileName "data/vexriscv_sim.cfg"
 getGdbCmdPath :: IO FilePath
 getGdbCmdPath = getDataFileName "data/vexriscv_gdb.cfg"
 
+getGdb :: HasCallStack => IO String
+getGdb = do
+  gdbMultiArch <- findExecutable "gdb-multiarch"
+  gdb <- findExecutable "gdb"
+  case gdbMultiArch <|> gdb of
+    Nothing -> fail "Neither gdb-multiarch nor gdb found in PATH"
+    Just x -> pure x
+
 expectLine :: Handle -> String -> Assertion
 expectLine h expected = do
   line <- hGetLine h
+  -- Uncomment for debugging:
+  -- hPutStr stderr "> "
+  -- hPutStrLn stderr line
```

Comment:
Would it make sense to have those lines behind a `when` maybe? The simulation runner has some top-level config, here there could be a `debug = False` maybe?

Reply:
Yeah we can add a flag to Tasty, no problem!